README.md

```diff
@@ -14,37 +18,54 @@ major release.
 Please join [gitter](https://gitter.im/census-instrumentation/Lobby) for help
 or feedback on this project.
 
-We encourage contributions. Use tags
-[up-for-grabs](https://github.com/census-instrumentation/opencensus-csharp/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs)
-and [good first
-issue](https://github.com/census-instrumentation/opencensus-csharp/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-to get started with the project. Follow [CONTRIBUTING](CONTRIBUTING.md) guide
-to report issues or submit a proposal.
+We encourage contributions. Use tags [up-for-grabs][up-for-grabs-issues] and
+[good first issue][good-first-issues] to get started with the project. Follow
+[CONTRIBUTING](CONTRIBUTING.md) guide to report issues or submit a proposal.
 
 ## OpenCensus Quickstart
 
 ### Using Zipkin exporter
 
 Configure Zipkin exporter to see traces in Zipkin UI.
 
-1. Get Zipkin using getting [started
-   guide](https://zipkin.io/pages/quickstart.html).
+1. Get Zipkin using [getting started guide][zipkin-get-started].
 2. Start `ZipkinTraceExporter` as below:
+3. See [sample][zipkin-sample] for example use.
 
 ``` csharp
 var exporter = new ZipkinTraceExporter(
   new ZipkinTraceExporterOptions() {
-    Endpoint = "https://<zipkin-server:9411>/api/v2/spans",
+    Endpoint = new Uri("https://<zipkin-server:9411>/api/v2/spans"),
     ServiceName = typeof(Program).Assembly.GetName().Name,
   },
   Tracing.ExportComponent);
 exporter.Start();
 
-var span = tracer.SpanBuilder("incoming request").SetSampler(Samplers.AlwaysSample).StartSpan();
+var span = tracer
+            .SpanBuilder("incoming request")
+            .SetSampler(Samplers.AlwaysSample)
+            .StartSpan();
+
 Thread.Sleep(TimeSpan.FromSeconds(1));
 span.End();
 ```
 
+### Using Application Insights exporter
+
+1. Create [Application Insights][ai-get-started] resource.
+  1. Set instrumentation key via configuration or global settings
+     (`TelemetryConfiguration.Active`).
```

Comment:
Please don't recommend using TelemetryConfiguration.Active. OC apps are likely .net core apps with dependency injection and using Active causes so much pain